Services/CO.CDP.OrganisationInformation.Persistence/DatabaseOrganisationRepository.cs

```diff
@@ -19,6 +19,14 @@ public void Dispose()
         return await context.Organisations.FirstOrDefaultAsync(t => t.Name == name);
     }
 
+    public async Task<IEnumerable<Organisation>> FindByUserUrn(string userUrn)
+    {
+        var person = await context.Persons
+            .Include(p => p.Organisations)
+            .FirstOrDefaultAsync(p => p.UserUrn == userUrn);
+        return person == null ? [] : person.Organisations;
```

Comment:
could be return `person?.Organisations ?? Enumerable.Empty<Organisation>();`?

Reply:
or even better `person?.Organisations ?? []`